scout/constants/query_terms.py

```diff
@@ -24,7 +24,7 @@
 # added as a top level '$and'.
 # Otherwise, primary criteria are added as a high level '$or' and all secondary criteria
 # are joined together with them as a single lower level '$and'.
-PRIMARY_CRITERIA = ["clinsig"]
+PRIMARY_CRITERIA = ["clinsig", "clinvar_tag"]
```

Comment:
Yes, you have a point. That is perhaps a better representation for that tag, and might make the exclusion a bit more intuitive.